vellum_ai.gemspec

```diff
@@ -4,7 +4,7 @@ require_relative "lib/gemconfig"
 
 Gem::Specification.new do |spec|
   spec.name = "vellum_ai"
-  spec.version = "v0.3.0"
+  spec.version = "0.3.0"
```

Comment:
Does ruby require that we don't prefix with the `v`?

Reply:
It's regular semvar! So yes, no v prefix, if it's a big change for you I can look to make the change within the generator itself